contracts/samples/callback/TokenCallbackHandler.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.12;
+
+/* solhint-disable no-empty-blocks */
+
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import "@openzeppelin/contracts/token/ERC777/IERC777Recipient.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+
+/**
+ * Token callback handler.
+ *   Handles supported tokens' callbacks, allowing account receiving these tokens.
+ */
+contract TokenCallbackHandler is IERC777Recipient, IERC721Receiver, IERC1155Receiver {
+    function tokensReceived(
+        address,
+        address,
+        address,
+        uint256,
+        bytes calldata,
+        bytes calldata
+    ) external pure override {
+    }
+
+    function onERC721Received(
+        address,
+        address,
+        uint256,
+        bytes calldata
+    ) external pure override returns (bytes4) {
+        return 0x150b7a02;
```

Comment:
is it possible at least to use the selector, instead of hard-code value
e.g.
```js
return IERC721Receiver.onERC721Received.selector;
```

Reply:
yeah, fixed.